flipperdevices/Flipper-Android-App
# Remote controls/design fixes

**Background**

This PR introduces fixes for design part of RemoteControls

**Changes**

- Title/Subtitle on remote screen
- Connection API for infrareds
- Progress loading on remote screen

**Test plan**

- Open setup screen, find remote
- See the progress now is shown on remote screen
